tools/aws_lab_setup/roles/control_node/tasks/main.yml

```diff
@@ -32,31 +32,161 @@
+      with_items:
+        - aws_lab_setup
+        - resources
+        - Vagrantfile
+        - README.md
+      tags:
+        - control_node
+        - control
 
-- name: Remove things that students don't need
-  file:
-    state: absent
-    path: ~{{ username }}/lightbulb/{{ item }}
-  with_items:
-    - aws_lab_setup
-    - resources
-    - Vagrantfile
-    - README.md
-  tags:
-    - control_node
-    - control
+    - name: Create lab inventory directory
+      file:
+        state: directory
+        path: /home/{{ username }}/lightbulb/lessons/lab_inventory
+      tags:
+        - control_node
+        - control
+
+    - name: Put student inventory in proper spot
+      copy:
+        src: ./{{ username }}-instances.txt
+        dest: "{{ control_node_inventory_path }}"
+        owner: "{{ username }}"
+        group: "{{ username }}"
+      when: username in inventory_hostname
+      tags:
+        - control_node
+        - control
+  when: networking is undefined or not networking
+
+- name: Ansible NETWORKING MODE
+  block:
+    - name: Clear out lightbulb directory spot (NETWORKING MODE)
+      file:
+        state: absent
+        path: /tmp/lightbulb
+
+    - name: Clone lightbulb repo (NETWORKING MODE)
+      git:
+        accept_hostkey: yes
+        clone: yes
+        dest: /tmp/lightbulb
+        repo: https://github.com/network-automation/lightbulb.git
+        force: yes
+      tags:
+        - lightbulb
+
+    - name: Move networking workshop folder to correct location  (NETWORKING MODE)
+      command: cp -r /tmp/lightbulb/workshops/networking/ /home/{{ username }}/networking-workshop
+      tags:
+        - lightbulb
+
+    - name: fix permissions of networking-workshop  (NETWORKING MODE)
+      file:
+        path: /home/{{ username }}/networking-workshop
+        owner: "{{ username }}"
+        group: "{{ username }}"
+        recurse: yes
+
+    - name: Create lab inventory directory (NETWORKING MODE)
+      file:
+        state: directory
+        path: /home/{{ username }}/networking-workshop/lab_inventory
+        owner: "{{ username }}"
+        group: "{{ username }}"
+      tags:
+        - control_node
+        - control
+
+    - name: Put student inventory in proper spot (NETWORKING MODE)
+      copy:
+        src: ./{{ username }}-instances.txt
+        dest: "/home/{{ username }}/networking-workshop/lab_inventory/hosts"
+        owner: "{{ username }}"
+        group: "{{ username }}"
+      when: username in inventory_hostname
+      tags:
+        - control_node
+        - control
+
+    - name: Put ssh-key in proper spot for student
+      copy:
+        src: ./aws-private.pem
+        dest: "/home/{{username}}/.ssh/"
+        owner: "{{ username }}"
+        group: "{{ username }}"
+      tags:
+        - testing
+
+    - name: Put ssh-key in proper for ec2-user
+      copy:
+        src: ./aws-private.pem
+        dest: "/home/ec2-user/.ssh/"
+        owner: "ec2-user"
```

Comment:
I'm pretty sure this break breaks centos support which doesn't have an ec2-user. @dfederlein am I mistaken?

Reply:
ah this is actually a super easy fix!  We already set the SSH user in the ec2_manage_instances role... I should have done this the first time.  I can just use {{ansible_user}} and it will work regardless of operating system!

Reply:
ok. then go for it.